Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -1935,6 +1935,112 @@ final class LayerTests: XCTestCase {
     XCTAssertEqual(transformerTensor.shape, transformerResult.shape)
   }
 
+  func testGroupNorm() {
+    // The expected values were computed using the following Python code:
+    // ```
+    // import tensorflow as tf
+    // import tensorflow_addons as tfa
+    // x = tf.reshape(tf.range(24, dtype=tf.float32), [2, 2, 1, 6])
+    // layer = tfa.layers.GroupNormalization(groups=2)
+    // with tf.GradientTape() as tape:
+    //     tape.watch(x)
+    //     y = layer(x)
+    //     z = tf.math.reduce_sum(tf.math.square(y))
+    // print(y, tape.gradient(z, [x] + layer.trainable_variables))
```

Comment:
Great job describing how the expected values were derived!  People often leave that information out.